acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/SignerConfigurationBuilder.java

```diff
@@ -66,8 +67,8 @@ public SignerConfigurationBuilder withServerTlsOptions(final TlsOptions serverTl
     return this;
   }
 
-  public SignerConfigurationBuilder withDownstreamTrustStore(final PkcsStoreConfig truststore) {
-    this.downstreamTrustStore = truststore;
+  public SignerConfigurationBuilder withDownstreamKnownServers(final File knownServersFile) {
```

Comment:
nit: sorry to be difficult - param has "file" postfix, member var doesn't ... which way to do you want to go? (happy either way, but would prefer to see it consistent)